src/Icons/ExtraPlush.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+
+interface Props {
+  width?: string | number
+  height?: string | number
+  fillColor?: string
+  [x: string]: unknown // for the rest property
+}
+
+const ExtraPlush: React.FunctionComponent<Props> = ({
+  width = '24',
+  height = '15',
+  fillColor = '#2D2926',
+  ...rest
+}) => (
+  <svg width="28" height="39" viewBox="0 0 28 39" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M4.33225 26.929V27.93H0.328247V21.4263H4.269V22.4273H1.38975V24.1158H4.01875V25.1168H1.38975V26.8685H4.3295L4.33225 26.929C4.33225 26.929 4.3295 26.929 4.33225 26.929Z"
+      fill={fillColor}
+    />
+    <path
+      d="M8.89997 27.9933L7.46172 25.6173L6.02347 27.996H4.77222L6.83747 24.6163L4.89872 21.4263H6.14997L7.46447 23.6153L8.77897 21.4263H10.0302L8.09147 24.6163L10.1567 27.996L8.89997 27.9933C8.89997 27.9933 8.89997 27.996 8.89997 27.9933Z"
+      fill={fillColor}
+    />
+    <path
+      d="M15.2195 22.4245H13.3412V27.93H12.277V22.4245H10.3987V21.4263H15.1535V22.4273C15.1535 22.4273 15.2195 22.4273 15.2195 22.4245Z"
+      fill={fillColor}
+    />
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```

Comment:
`fill-rule` and `clip-rule` need to be `fillRule` and `clipRule` for react. Change on every path below